src/main/java/com/bunq/sdk/model/generated/CustomerLimit.java

```diff
@@ -0,0 +1,104 @@
+package com.bunq.sdk.model.generated;
+
+import com.bunq.sdk.context.ApiContext;
+import com.bunq.sdk.http.ApiClient;
+import com.bunq.sdk.http.BunqResponse;
+import com.bunq.sdk.http.BunqResponseRaw;
+import com.bunq.sdk.model.BunqModel;
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Show the limits for the authenticated user.
+ */
+public class CustomerLimit extends BunqModel {
+
+  /**
+   * Endpoint constants.
+   */
+  private static final String ENDPOINT_URL_LISTING = "user/%s/limit";
+
+  /**
+   * Object type.
+   */
+  private static final String OBJECT_TYPE = "CustomerLimit";
+
+  /**
+   * The limit of monetary accounts.
```

Comment:
Shouldn't  it be the limit `for` instead of `of` ?

Reply:
generated